README.md

```diff
@@ -71,6 +71,14 @@ Considering the balance between correctness and performance, `sse2neon` recogniz
 
 The above are turned off by default, and you should define the corresponding macro(s) as `1` before including `sse2neon.h` if you need the precise implementations.
 
+## Add New Intrinsic
```

Comment:
Move this sub-section to the place between "Run Built-in Test suite" and "Coding Convention."

Reply:
Done.